workloads/files/workload-network-script-cm.yml

```diff
@@ -0,0 +1,500 @@
+      if [ "${NETWORK_TEST_SERVICE}" == "true" ]; then
+        config_prefix="S2S"
+        PAIR_COUNT=${pair_count} envsubst < /root/workload/s2s-network.yaml.template > /tmp/network.yaml
+        cp /root/workload/svc-template.yaml /tmp/svc-template.yaml
+        # Append required ports to service template
+        for i in `seq ${NETWORK_TEST_UPERF_PORT_RANGE}`
+        do
+          echo -e "    - name: tcp-${i}\n      port: ${i}\n      protocol: TCP" >> /tmp/svc-template.yaml
+          echo -e "    - name: udp-${i}\n      port: ${i}\n      protocol: UDP" >> /tmp/svc-template.yaml
+        done
+      else
+        PAIR_COUNT=${pair_count} envsubst < /root/workload/p2p-network.yaml.template > /tmp/network.yaml
+      fi
+
+      VIPERCONFIG=/tmp/network.yaml openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster should load the cluster [Suite:openshift]"
+
+      # Check if clients and servers are ready
+      for rc_num in `seq 0 $((${pair_count} - 1))`
+      do
+        retries=0
+        while [ $retries -le 60 ] ; do
+          client_ready=`oc get replicationcontroller/uperf-client-${rc_num} -o json -n network0 | jq '.status.replicas==.status.availableReplicas'`
+          if [ "${client_ready}" == "true" ]; then
+            log "uperf-client-${rc_num} is ready"
+            break
+          else
+            log "Waiting for uperf-client-${rc_num} to be ready"
+            sleep 2
+          fi
+          retries=$[$retries + 1]
+        done
+        if [ "${client_ready}" == "false" ]; then
+          log "uperf-client-${rc_num} failed to be ready on time"
+          exit 1
+        fi
+        retries=0
+        while [ $retries -le 60 ] ; do
+          server_ready=`oc get replicationcontroller/uperf-server-${rc_num} -o json -n network0 | jq '.status.replicas==.status.availableReplicas'`
+          if [ "${server_ready}" == "true" ]; then
+            log "uperf-server-${rc_num} is ready"
+            break
+          else
+            log "Waiting for uperf-server-${rc_num} to be ready"
+            sleep 2
+          fi
+          retries=$[$retries + 1]
+        done
+        if [ "${server_ready}" == "false" ]; then
+          log "uperf-server-${rc_num} failed to be ready on time"
+          exit 1
+        fi
+      done
+
+      if [ "${NETWORK_TEST_SERVICE}" == "true" ]; then
+        clients=`oc get svc --output=json -n network0 --selector="name=client" | jq -r '[.items[].spec.clusterIP] | join(",")'`
+        servers=`oc get svc --output=json -n network0 --selector="name=server" | jq -r '[.items[].spec.clusterIP] | join(",")'`
+      else
+        clients=`oc get pods --output=json -n network0 --selector="name=client" | jq -r '[.items[].status.podIP] | join(",")'`
+        servers=`oc get pods --output=json -n network0 --selector="name=server" | jq -r '[.items[].status.podIP] | join(",")'`
+      fi
+
+      mv ~/.ssh/config ~/.ssh/config.bak || true
```

Comment:
@jmencak Let me know if you think we should change this usage of `|| true` it allows privileged and unprivileged execution of this test.  I figured you would be best to review it.

Reply:
Yeah, the joys of `set -e`.  I'd prefer to have a home directory always writable so that this doesn't fail and you don't need `|| true` or similar.  That's why I'm changing `HOME` for my tests.  Didn't run this test yet, so not sure if that's the same issue I had before I changed my `HOME`.

Reply:
Yeah in this case /.ssh/config does not exist and won't exist as we do not put it into the container image.  Even if we did it would be unclear what user should own it as it is the 'unprivileged' execution that hits it. Privileged execution has a $HOME of `/root` and thus find `/root/.ssh/config`.  My opinion is this is that this is an acceptable "1-line" solution.  Another option would be just writing an if statement however that would add a few more lines to each of the two statements that need `|| true`  I like to think that this is an acceptable hack considering it is not complex and that it does the job as needed with less lines.